wenet/transformer/asr_model.py

```diff
@@ -332,7 +332,7 @@ def _ctc_prefix_beam_search(
                                reverse=True)
             cur_hyps = next_hyps[:beam_size]
         hyps = [x[0] for x in cur_hyps]
-        return hyps, encoder_out
+        return hyps, encoder_out, cur_hyps
```

Comment:
hyps shares the same info with cur_hyps, it's better we return hyps with it's corresponding total score here. The return is like List[(hyp, score)], encoder_out.